doc-content/shared-kie-docs/src/main/asciidoc/Install/testing-clustering-environment-proc.adoc

```diff
@@ -0,0 +1,122 @@
+[id='testing-clustering-environment-proc_{context}']
+= Testing that your infrastructure meets the minimum hardware and performance requirements for a viable high availability (HA) on-premises environment.
+
+When creating a production-ready high availability (HA) on-premises infrastructure for {CENTRAL}, you must ensure it meets the minimum hardware and performance requirements for a viable HA environment, and to mitigate performance related issues. This section describes the environment elements, an example environment, and how to test the performance.
+
+There are four main components; {CENTRAL}, the message system (AMQ), the indexing server ({DATAGRID}), and a shared file system (NFS/GlusterFS/Ceph).
+
+.Prerequisites
+
+* A network environment of at least 3 nodes is configured with the following layout:
++
+Node 1: {CENTRAL}
++
+Node 2: {CENTRAL}
++
+Node 3: {CENTRAL}, {DATAGRID}, and NFS
```

Comment:
{Central} is not in node 3, but AMQ

Reply:
Done, changed to "Node 3: AMQ, {DATAGRID}, and NFS"